### PR TITLE
Updated docs to fix typo and update grid status

### PR DESCRIPTION
**Pull request by aribornstein**
## What does this PR do?

Updated docs to fix typo and update grid status

### CODE REVIEW

File: docs/source/starter/new-project.rst [hunk 1]
```diff
@@ -761,9 +761,9 @@ Or read our :doc:`Guide <../starter/introduction_guide>` to learn more!
 
 Grid AI
 =======
-Grid AI is our native solution for large scale training and tuning on the cloud provider of your choice.
+Grid AI is our native solution for large scale training and tuning on the cloud.
 
-`Click here to request early-access <https://www.grid.ai/>`_.
+`Get started for free with you Github or Google Account here <https://www.grid.ai/>`_.
```
Comment (ananthsub) [inline docs/source/starter/new-project.rst:766]:
```suggestion
`Get started for free with your Github or Google Account here <https://www.grid.ai/>`_.
```

File: docs/source/starter/new-project.rst [hunk 2]
```diff
@@ -741,7 +741,7 @@ Lightning has many tools for debugging. Here is an example of just a few of them
 ---------------
 
 ********************
-Other coool features
+Other cool features
```
Comment (awaelchli) [inline docs/source/starter/new-project.rst:744]: coool with three o's is kinda cool

File: docs/source/starter/new-project.rst (hunk 1 above)
Comment (awaelchli) [inline docs/source/starter/new-project.rst:766]:
```suggestion
`Get started for free with you GitHub or Google Account here <https://www.grid.ai/>`_.
```

**Review by awaelchli — APPROVED** [review]

**Review by carmocca — APPROVED** [review]

File: docs/source/starter/new-project.rst [hunk 3]
```diff
@@ -741,7 +741,7 @@ Lightning has many tools for debugging. Here is an example of just a few of them
 ---------------
 
 ********************
```
Comment (kaushikb11) [inline docs/source/starter/new-project.rst:743]:
```suggestion
*******************
```

File: docs/source/starter/new-project.rst [hunk 4]
```diff
@@ -741,7 +741,7 @@ Lightning has many tools for debugging. Here is an example of just a few of them
...
-Other coool features
+Other cool features
 ********************
```
Comment (kaushikb11) [inline docs/source/starter/new-project.rst:745]:
```suggestion
*******************
```

**Review by SeanNaren — APPROVED** [review]

**Pull request merged**